src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  numOfCategories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {
+    updateStatus: (state, action) => {
+      state.status = action.payload;
+    },
```

Comment:
- [ ] Well, your integration of a reducer for the `status` within the `categoriesSlice` is commendable. However, aligning with the specific project prerequisites, the reducer ought to evaluate the `status` and consistently yield the value "Under construction". To accomplish this, it's imperative to establish a conditional check within the reducer's logic. Here's an illustrative example for your guidance:

```javascript
updateStatus: (state, action) => {
  if (action.payload === "unsuccess") {
    state.numOfCategories = "Under construction";
  } else {
    state.numOfCategories = state.numOfCategories;
  }
},
```
![image](https://github.com/NewtonOmbese/bookstore/assets/55840999/f6f5f24e-3a21-4b2d-b06d-fe301803538e)

By incorporating this conditional construct, you're ensuring that the `status` reflects the mandated "Under construction" output when appropriate. This level of precision underscores your commitment to adhering to project requisites. Kudos on your dedication! 🌟📝🔧